.travis.yml

```diff
@@ -1,17 +1,11 @@
+# Auto-generated !!! Do not edit it manually
+# use ci-master https://github.com/metanorma/metanorma-build-scripts
 language: ruby
 dist: xenial
 env:
   global:
+    - JAVA_OPTS="java.awt.headless=true"
```

Comment:
@CAMOBAP795 do we really need this? Probably not.

Reply:
I have moved all docs repos to the same CI config https://github.com/metanorma/metanorma-build-scripts/commit/fbe9a7ffd1a246b0ea8aad4a5aab51ed382b23a6

If it doesn't make sense for all repos let me know

Reply:
@CAMOBAP795 I don't think we need it -- could you check if it's OK, and update the branches if it passes? Thanks!

Reply:
@ronaldtse make sens to keep it http://plantuml.com/faq#22

Reply:
OK makes sense.